cmd/sqlflowserver/main_test.go

```diff
@@ -114,6 +115,40 @@ func ParseRow(stream pb.SQLFlow_RunClient) ([]string, [][]*any.Any) {
 	return columns, rows
 }
 
+func prepareTestData(dbStr string) error {
+	// popularize test data
+	testDB, err := sql.Open(dbStr)
+	if err != nil {
+		return err
+	}
+	_, err = testDB.Exec("CREATE DATABASE IF NOT EXISTS iris;")
```

Comment:
Could we move `CREATE DATABASE IF NOT EXISTS iris` in `sql/testdata/iris.sql`, other than in this .go file?